importtest.py

```diff
@@ -0,0 +1,7 @@
+from bpython.importcompletion import find_modules
```

Comment:
If we want to keep this, you should move it to the tests folder and make it an automated test, along with these other folders needed to test this. However we probably don't want to check symlinks into version control. If you make this an automated test, I'd use functions like `os.mkdir` and `os.symlink` to build these structures in the test, run the test, then clean the folders and files up.

Reply:
I agree, this should be part of the test suite, but `tempfile.TemporaryDirectory` and `os.symlink` should be used to setup the test.